README.md

```diff
@@ -128,8 +128,13 @@ While periscope uses shiny dashboard variables for styling in code and via the *
 
 ### Installation
```

Comment:
I would suggest moving this at the start of the readme file so it is easier to find maybe right after the overview?

Reply:
Thanks @agenius-jasmine - I will try to update ReadME before submitting to CRAN or at the nearest chance